jobs/route_registrar/spec

```diff
@@ -36,31 +37,52 @@ properties:
           The IP is determined automatically from the host on which route-registrar is run.
-        tls_port (required, integer): Either `port` or `tls_port` are required; if both are provided, Gorouter will prefer tls_port.
-          Requests for associated URIs will be forwarded over TLS by the router to this port. 
+        tls_port (required, integer, for http routes): Either `port` or `tls_port` are required; if both are provided, Gorouter will prefer tls_port.
+          Requests for associated URIs will be forwarded over TLS by the router to this port.
           The IP is determined automatically from the host on which route-registrar is run.
-        server_cert_domain_san (conditional, string): Required if tls_port is present. 
+        server_cert_domain_san (conditional, string, for http routes): Required if tls_port is present.
           Gorouter will validate that the TLS certificate presented by the destination host contains this as a Subject Alternative Name (SAN).
-        registration_interval (required, string): Interval between heartbeated route registrations
+        registration_interval (required, string, for all routes): Interval between heartbeated route registrations
           (e.g. 10s). It must parse to a positive time duration i.e. "-5s" is not permitted.
-        tags (optional, array of objects): Arbitrary key-value pairs emitted with metrics to support
+        tags (optional, array of objects, for http routes): Arbitrary key-value pairs emitted with metrics to support
           filtering of metrics
-        prepend_instance_index (optional, boolean): When set to true the values in `uris` will be
+        prepend_instance_index (optional, boolean, for http routes): When set to true the values in `uris` will be
           prepended with the instance index. e.g. 'some-uri.system-domain.com' will become '0-some-uri.system-domain.com' on the instance with index 0, and '2-some-url.system-domain.com' on the instance with index 2. When this value is enabled, each instance will register its own, unique, set of uris. To additionally continue to register these original uris, create another route with the same uris and set 'prepend_instance_index' to false (or omit the key entirely).
-        health_check (optional, object): Script executed on frequency of `registration_interval`. If
+        health_check (optional, object, for all routes): Script executed on frequency of `registration_interval`. If
           healthcheck script exits with success, route registration heartbeat is sent. If script exits
-          with error, the route is unregistered. 
+          with error, the route is unregistered.
+        router_group (required, string, for tcp routes): Name of the router group to which the TCP route should be added.
+        backend_host (required, string, for tcp routes): DNS or IP address of the backend server that the TCP route should be routed to.
```

Comment:
Do you need to register a TCP route for a host other than the one on which route_registrar runs? If not, this is not necessary; route_registrar automatically detects the IP of the VM.

Reply:
If you do need to specify the backend host, you can leverage the existing top-level `host` property: https://github.com/cloudfoundry/routing-release/blob/develop/jobs/route_registrar/spec#L35-L37